src/reducers/map/create/success.ts

```diff
@@ -14,31 +14,15 @@ var invariantArgs: InvariantsBaseArgs = {
 export default function success(config: Config, current: Map<any>, addedRecord: any, clientGenKey?: string): Map<any> {
 	invariants(invariantArgs, config, current, addedRecord)
 
-	var key = config.key
-	var done = false
-	var addedRecordKey = addedRecord[key]
-
-	// Update existing records
-	var updatedCollection = r.map((existingRecord) => {
-		var recordKey = existingRecord[key]
-		if (recordKey == null) throw new Error('Expected record to have ' + key)
```

Comment:
Come to think of it.... I'm wondering why the "it throws when record doesnt have an id" test didn't start failing when I removed this. Hmmmm